### PR TITLE
BAN-91: Zen input with slots

### PR DESCRIPTION
**Pull request by f-jauregui**
[Jira ticket](https://reciprocitylabs.atlassian.net/browse/BAN-91)

### CODE REVIEW

**Review by mihaelpalfi — APPROVED** [review]
Great work with the slots!

File: src/components/zen-input/zen-input.scss [hunk 1]
```diff
@@ -2,12 +2,57 @@
...
 }
 
-input.input-control {
-  @include zenInput;
-  @include noselect;
+:host(:hover) {
+  border-color: $color-blue-700;
+}
+
+:host(.has-focus) {
```
Comment (chavlji) [inline src/components/zen-input/zen-input.scss:14]: I think it's great that you're using host to paint the border. I just don't like that we can't use the old mixin for that. We should take a look into that later, to unify it somehow. Shall we create a ticket for that?

**Review by chavlji — CHANGES_REQUESTED** [review]
Looks good! We'll need public method `toggleFocused` exposed so we can turn it on/off when dropdown is un/focused.

File: src/components/zen-input/zen-input.scss (hunk 1 above)
Comment (f-jauregui) [reply src/components/zen-input/zen-input.scss:14]: I don't think so, if we move forward with Tailwind these mixins won't be needed. I won't fix something that will be replaced sooner than later.

**Review by chavlji — APPROVED** [review]
Great! 🚀

**Pull request merged**